components/encrypt-video.jsx

```diff
@@ -7,6 +8,7 @@ module.exports = React.createClass({
       this.props.setPageState({
         videoIsPaused: false
       });
+      ga.event({category: "Video", action: "Video started"});
     });
     this.refs.video.addEventListener("webkitendfullscreen", () => {
```

Comment:
We could probably use a track here, and maybe there's a way to capture the current timecode from the video when it gets triggered...